WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -1,28 +1,56 @@
 package org.wordpress.android.ui.stories
 
 import android.os.Bundle
-import android.widget.Toast
 import com.wordpress.stories.compose.ComposeLoopFrameActivity
 import com.wordpress.stories.compose.MediaPickerProvider
 import com.wordpress.stories.compose.SnackbarProvider
+import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.ActivityLauncher
+import org.wordpress.android.ui.RequestCodes
+import org.wordpress.android.ui.media.MediaBrowserType
+import org.wordpress.android.ui.photopicker.PhotoPickerActivity
 
 class StoryComposerActivity : ComposeLoopFrameActivity(), SnackbarProvider, MediaPickerProvider {
+    private var site: SiteModel? = null
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setSnackbarProvider(this)
         setMediaPickerProvider(this)
+
+        if (savedInstanceState == null) {
```

Comment:
We do this all over the app, so feel free to keep it as is. 

Storying the whole model into a bundle is considered a bad practice (not mentioning that we are using serializable and not parcelable:P). We might want to consider passing just an id of the site in the Intent. ViewModel could load this site from the database and cache it. Wdyt?

Reply:
That's a good idea and I recall it having come up in the past - I purposedly tried to be a "good citizen" with the target codebase and write code that looks along the lines of what already existed :) , but you have a point in this. I have been thinking for a while and I think the SiteModel itself is not as convoluted as say a PostModel can be, so I think it'd be preferable to keep the code consistent for now and bring the Site id passing improvements as part of a different refactor. Thanks for the suggestion! 🙇

Reply:
I believe we already use both approaches in the code base. Having said that I'm completely fine keeping it as is, since it's a very minor "issue". ;)